### PR TITLE
Feature/cplp 1894/deactivation

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**

### CODE REVIEW

File: cx-portal-shared-components/src/components/content/Cards/ContentCards.stories.tsx [hunk 1]
```diff
@@ -107,4 +115,7 @@ ContentCards.args = {
   imageSize: 'medium',
   imageShape: 'round',
   addButtonClicked: false,
+  subMenu: true,
+  submenuOptions: submenuOptions,
+  tooltipText: 'Action is pending',
```
Comment (manojava-gk) [inline cx-portal-shared-components/src/components/content/Cards/ContentCards.stories.tsx:120]: Better this to move to locale files

File: cx-portal/src/components/pages/AppOverview/Deactivate.tsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
+/********************************************************************************
+ * Copyright (c) 2021,2022 Mercedes-Benz Group AG and BMW Group AG
```
Comment (manojava-gk) [inline cx-portal/src/components/pages/AppOverview/Deactivate.tsx:2]: as you are creating this file, this should be BMW's template

File: cx-portal-shared-components/src/components/content/Cards/ContentCards.stories.tsx (hunk 1 above)
Comment (nidhigarg-bmw) [reply cx-portal-shared-components/src/components/content/Cards/ContentCards.stories.tsx:120]: @manojava-gk  This is the static text in stories file.

File: cx-portal/src/components/pages/AppOverview/Deactivate.tsx (hunk 2 above)
Comment (nidhigarg-bmw) [reply cx-portal/src/components/pages/AppOverview/Deactivate.tsx]: @manojava-gk  Made the change :)

**Review by manojava-gk — APPROVED** [review]
lftm, resolve conflicts

**Review by manojava-gk — APPROVED** [review]
looks good

**Pull request merged**